tests/baselines/exportDeclarations.baseline.txt

```diff
@@ -40,23 +43,23 @@ Grammar: TypeScript.tmLanguage
                      source.ts punctuation.terminator.statement.ts
 >export * from "AnotherModule";
  ^^^^^^
- source.ts meta.export.ts keyword.control.export.ts
+ source.ts storage.modifier.ts
        ^
-       source.ts meta.export.ts
+       source.ts
         ^
-        source.ts meta.export.ts constant.language.import-export-all.ts
+        source.ts keyword.operator.arithmetic.ts
```

Comment:
This looks incorrect change

Reply:
Forgot to update baselines after I fixed this. Pushed now.